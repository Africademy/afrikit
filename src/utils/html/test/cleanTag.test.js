// // @flow
// import React from 'react';
// import expect from 'expect';
// import { create as render } from 'react-test-renderer';
// import tag from '../cleanTag';

// describe('cleanElement', () => {
//   it('renders', () => {
//     const json = render(React.createElement(tag)).toJSON();
//     expect(json.type).toEqual('div');
//     expect(json).toMatchSnapshot();
//   });

//   test('omits props', () => {
//     const json = render(
//       React.createElement(tag, {
//         id: 'hello',
//         m: 2,
//         px: 3,
//         color: 'blue',
//       }),
//     ).toJSON();
//     expect(json.props.m).toBeUndefined();
//     expect(json.props.px).toBeUndefined();
//     expect(json.props.blue).toBeUndefined();
//     expect(json.props.id).toEqual('hello');
//   });

//   test('exports html tags', () => {
//     const h1 = render(React.createElement(tag.h1)).toJSON();
//     const header = render(React.createElement(tag.header)).toJSON();
//     expect(h1.type).toEqual('h1');
//     expect(header.type).toEqual('header');
//   });

//   test('exported html tags only omit blacklisted props', () => {
//     const json = render(
//       React.createElement(tag.h1, {
//         id: 'hello',
//         m: 2,
//         px: 3,
//         color: 'blue',
//       }),
//     ).toJSON();
//     expect(json.props.m).toBeUndefined();
//     expect(json.props.px).toBeUndefined();
//     expect(json.props.blue).toBeUndefined();
//     expect(json.props.id).toEqual('hello');
//   });
// });
