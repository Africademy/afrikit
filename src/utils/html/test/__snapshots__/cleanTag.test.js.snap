// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cleanElement renders 1`] = `<div />`;
